Formula/docbook-xsl.rb

```diff
@@ -60,6 +60,7 @@ def install
       doc.install "doc" => "reference"
     end
 
+    chmod 0755, "#{prefix}/docbook-xsl/epub/bin/dbtoepub"
```

Comment:
Why is this needed if we wrap it in the line below?

Reply:
The wrapper script will fail if the thing it calls it not executable:

    /usr/local/bin/dbtoepub: line 2: /usr/local/Cellar/docbook-xsl/1.79.2_1/docbook-xsl/epub/bin/dbtoepub: Permission denied
    /usr/local/bin/dbtoepub: line 2: exec: /usr/local/Cellar/docbook-xsl/1.79.2_1/docbook-xsl/epub/bin/dbtoepub: cannot execute: Undefined error: 0

Reply:
Actually, if this should be executable it should be fixed upstream.